src/react/components/styleform/PointTab.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { SvgRadioButton } from 'oskari-ui';
+import { Form, Row } from 'antd';
+
+export const PointTab = (props) => {
+    return (
+        <>
+            <Row>
+                <Form.Item
+                    { ...props.formLayout }
+                    name='image'
+                    label='Ikoni'
+                    onChange={ (event) => props.onChangeCallback('image.shape', event.target.value) }
+                >
+                    <SvgRadioButton options={ props.markers } defaultValue={ props.styleSettings.image.shape } />
```

Comment:
Here we do want to pass the markers as props so don't change this one (line styles and area fill styles can be baked in the form component).